src/framework/renderer/js/RendererUtilities.js

```diff
@@ -307,16 +307,17 @@ fluid_3_0_0 = fluid_3_0_0 || {};
     fluid.renderer.selection.inputs = function (options, container, key, config) {
         fluid.expect("Selection to inputs expander", options, ["selectID", "inputID", "labelID", "rowID"]);
         var selection = config.expander(options.tree);
+        var expandedOpts = config.expandLight(options);
```

Comment:
This is dangerous/inefficient since the options will contain some material that should not be fed through expandLight - primarily the tree, which will contain strings whose syntax could be confused with IoC references. I suggest making a copy using http://docs.fluidproject.org/infusion/development/CoreAPI.html#fluid-censorkeys-tocensor-keys- to filter out tree and any other similar dangerous material. This would be a great opportunity to add a docs comment to this function to explain what is the structure and meaning of the accepted arguments.

It appears that the closest thing we have resembling docs is on this page - https://wiki.fluidproject.org/display/docs/Renderer+Component+Tree+Expanders#RendererComponentTreeExpanders-SelectionToInputsExpander